docs/pages/eas-update/introduction.md

```diff
@@ -2,6 +2,10 @@
 title: Introduction
 ---
 
+import Head from '~/components/Head'
+
+<Head title="EAS Update Introduction - Expo Documentation" />
```

Comment:
alternative suggestion: Introduction to EAS Update

I think it reads better but feel free to go with your preference